src/redux/selectors/selectors.js

```diff
@@ -0,0 +1,64 @@
+import { createSelector } from 'reselect'
+
+import { getDailyPage, getImagesGalleryPage } from 'redux/selectors/index'
```

Comment:
when you have a **/index** file somewhere and you try to import some stuff from it, you could always omit it as that's the way webpack works, it searches for an index.js file when you import something from a directory. use this:
```
import { getDailyPage, getImagesGalleryPage } from 'redux/selectors'
```